src/common/vitals/cumulative-layout-shift.js

```diff
@@ -7,7 +7,7 @@ export const cumulativeLayoutShift = new VitalMetric(VITAL_NAMES.CUMULATIVE_LAYO
 
 if (isBrowserScope) {
   onCLS(({ value, entries }) => {
-    if (value === cumulativeLayoutShift.value.current) return
+    if (cumulativeLayoutShift.roundingMethod(value) === cumulativeLayoutShift.value.current) return
```

Comment:
Storing the unrounded value would be better because current: 83.3 and new: 83.4 would make this a false positive under flooring.

Reply:
(Vital numbers have a bigger granularity so it's easy to lose accuracy, unlike performance domhighrestimestamp numbers.)

Reply:
I think you also flatten the new `entries` as an array in other commit, but if you want to be able to tell which set of entries is responsible for yielding which value point, they would have to be appended as-is (2-d array).

Reply:
Guess it depends on your definition. Checking that the rounded values are equal is important because it's the rounded value that gets reported (and we don't want to re-emit it)